website/content/labs/lab4.md

```diff
@@ -106,35 +105,36 @@ Runtime errors make the program crash while it is running, and sometimes they ar
 
 `frame n` (where `n` is the frame number from the backtrace) allows you to go to the function in that frame and see what line in that particular function caused the error.
 
-Compile `task3.c` and run it. You will segfault. Now it’s time to debug! There are three errors in this file. For each error, answer the following questions in `task3.txt`. It is a good idea to fix one error before moving onto the next.
+Compile `task3.c` and run it. You will segfault. Now it’s time to debug! There are three errors in this file. For each error, answer the following questions in `task3.txt`. It is a good idea to fix one error before moving on to the next.
```

Comment:
It may be helpful to remind them to use the -g flag (Even though it was just last task I forgot it)

Reply:
Fixed.